### ISSUE TITLE
Multiple settings on same line only converts first instance

### ISSUE THREAD

**Issue by cshold** [bit3725/postcss-shopify-settings-variables#8]
Writing this:

```
font-family: $(type_header_font_family.family), $(type_header_font_family.fallback_families);
```

I would expect:
```
font-family: {{ settings.type_header_font_family.family }}, {{ settings.type_header_font_family.fallback_families }};
```

But get this:
```
font-family: {{ settings.type_header_font_family.family }}, $(type_header_font_family.fallback_families);
```

Only the first $(variable) is parsed.

**Comment by coppinger** [comment]
Any updates on this cshold? Doesn't seem like a deal-breaker considering the utility this plugin would bring, but it would be great to have it working as intended.

**Comment by cshold** [comment]
Nope

**Comment by coppinger** [comment]
> Nope

Did you end up using this at all? I'm keen to use it on a project starting this week, as I'd like to utilise Tailwind CSS and it requires PostCSS. The inability to set variables to schema values would be a deal-breaker.

**Comment by cshold** [comment]
Yes, but organized my variables differently. For most Shopify variables I started using CSS variables in `:root` in theme.liquid. Takes the need out of this plugin mostly so I'll likely move away from it soon.

**Comment by coppinger** [comment]
Gotcha, thanks for letting me know - I think I'll probably go down that path too.

I've been meaning to have a catchup with some other Shopify theme dev's such as yourself for some time, as I'd love to share workflows and insights. If you might be interested in a short Zoom call at some point, drop me a message or email me at charles.coppinger@gmail.com

Cheers!

**Comment by coppinger** [comment]
Apologies Carson, I've just done my homework and realised you're a bit of a legend when it comes to Shopify theme development - my invite for some knowledge sharing still stands, but I completely understand if you're too busy/that's not something you're interested in doing.

Safe to say, the flow of knowledge would likely be in one direction!

**Comment by cshold** [comment]
Hah, you humble me. I'd say check out the [ecommTalk Slack](https://ecommtalk.com/). Lots of good people in there with plenty of knowledge.

**Comment by bit3725** [comment]
Hey guys, I just pushed a update to solve this issue. Could you please help to test if version 0.4.0 solves your problems if you still need this plugin?
